### DISCUSSION TITLE
No data in the performance summary or Performance tabs

### DISCUSSION THREAD

**Discussion by dennisa72295** [trimble-oss/dba-dash#889] [Q&A]
How do I turn it on so that I am collecting performance data?

**Comment by dennisa72295** [comment]
I found it.  I had No WMI checked on the config tool

**Reply by DavidWiseman** [reply]
The WMI collections are just used for things like drive space, computer & driver info - no performance data.  [More info here](https://dbadash.com/docs/help/wmi/)

To pick up new instances in the GUI you either need to click Filter\Clear All or right-click the root node in the tree and click Refresh.  I think that might have been the issue.

Glad it's sorted anyway! :-)
